cypress/e2e/API/api.cy.js

```diff
@@ -0,0 +1,14 @@
+/// <reference types="Cypress" />
+
+//API Test Suit
+describe('API Tests', () => {
+
+    it('Recupera clientes com sucesso', () => {
+        //Realiza uma request para a p�gina e verifica se a resposta � 200
+        cy.request('https://engage-sphere.vercel.app').should(function (response) {
```

Comment:
Recomendo desestrutura o `response` direto aqui, evitando fazer `const { status } = response` logo abaixo.

Em vez disso, poderia ser:

```suggestion
        cy.request('https://engage-sphere.vercel.app').should(function ({ status }) {
```

Reply:
Para aprender mais sobre o assunto, assista a seguinte live que fiz no Canal TAT no YouTube: [JavaScript para QAs #8 - Atribuição via Desestruturação](https://youtube.com/live/yW1JQoz6PXU).